doc/cygnus-ngsi/flume_extensions_catalogue/orion_rest_handler.md

```diff
@@ -131,8 +131,8 @@ As said, Flume events are not much more different than the above representation:
 | Parameter | Mandatory | Default value | Comments |
 |---|---|---|---|
 | notification\_target | no | `notify/` | Any other configured value must start with `/`. |
-| default\_service | no | `default` || 
-| default\_service\_path | no | `/` | `/` is the root service path (also know as root subservice). Any other configured value must start with `/`. |
+| default\_service | no | `default` | Alphanumerics and underscores are only accepted. | 
+| default\_service\_path | no | `/` | `/` is the root service path (also know as root subservice). Any other configured value must start with `/`. Apart from the initial slash, alphanumerics and underscores are onlyu accepted. |
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/987/commits/9ef022944a8cd0eacfe06d813b1c25b953da8cbc